parsely-parsely-page.php

```diff
@@ -4,6 +4,7 @@
    <script type="application/ld+json">
    <?php echo json_encode($parselyPage); ?>
    </script>
+   <meta name='parsely-post-id' content='<?php echo $postId ?>' />
```

Comment:
This errors out on pages with no $postId, likely because pages / section etc. don't have postIds but can have headlines, thus fulfilling the if condition above. Might be worth checking if $postid is set and is not empty before echoing this